### PR TITLE
LibreOffice update

### PR DESCRIPTION
**Pull request by d1nuc0m**
- Divided LibreOffice in Fresh and Still branches 
- Updated Still to latest version working on ReactOS (tested on ROS 0.14.3 on VirtualBox 6.1.x)
- Updated URLs and descriptions

### CODE REVIEW

**Review by learn-more — COMMENTED** [review]
Thanks for taking the time to update this!

I think that adding 'fresh' to the name is only going to confuse people, since this is not a very common term for software.

File: libreofficefresh.txt [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [Section]
-Name = LibreOffice
+Name = LibreOffice Fresh
```
[the remark this answers is not in the document]
Comment (d1nuc0m) [reply libreofficefresh.txt:2]: It is a LibreOffice convention, LibreOffice Fresh is the more up-to-date branch (in this moment it is 7.2.x), more functions but more prone to bugs; LibreOffice Still is a more stable branch (in this moment it is 7.1.x), slightly behind Fresh version, but more robust.
I have checked the latest Fresh and Still branches that work on ReactOS, and are respectively 5.4.x and 5.3.x

File: libreofficestill.txt [hunk 2]
```diff
@@ -1,26 +1,26 @@
 [Section]
-Name = LibreOffice
-Version = 5.1.1.3
+Name = LibreOffice Still
+Version = 5.3.7.2
 License = LGPL
-Description = A powerful and open source office suite. It has been forked from OpenOffice.
-Size = 211.5 MiB
+Description = LibreOffice is a free and powerful office suite, and a successor to OpenOffice.org (commonly known as OpenOffice).
+Size = 210 MiB
```
[the remark this answers is not in the document]
Comment (d1nuc0m) [reply libreofficestill.txt:6]: Thank you, where can I find a complete list of available and required fields? The wiki and this repository seem to be missing some details

[the remark this answers is not in the document]
Comment (binarymaster) [reply libreofficestill.txt:6]: See https://reactos.org/wiki/RAPPS

File: libreofficefresh.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (d1nuc0m) [reply libreofficefresh.txt:2]: @learn-more Fresh/Still difference has also been discussed [on LibreOffice forums](https://ask.libreoffice.org/t/what-is-the-difference-between-libreofficefresh-and-libreofficestill/18684), same for [compatibility with x86 XP/2003](https://ask.libreoffice.org/t/whats-latest-ver-i-can-use-on-my-32bit-xp-sp3-machines/24434/2)

**Review by JoachimHenze — CHANGES_REQUESTED** [review]
Requested change: We currently have 2 LibreOffice versions already in ros.
**We should delete one, and keep (+update) only one of them.**

**Ofc only after testing whether the new version works well on 2k3sp2 and in ros. Please post screenshots that prove both for the version that you suggest here.**

I do also contribute something: the proof by previous tests that both of our outdated existing versions do run in a comparable manner: search for "LibreOffice" in https://reactos.org/wiki/Tests_for_0.4.14
I spotted 16bugs quickly in both. They do behave almost identically. A waste of time to always test 2 versions for each release.

**Comment by d1nuc0m** [comment]
> 
> 
> Requested change: We currently have 2 LibreOffice versions already in ros.
> **We should delete one, and keep (+update) only one of them.**
> 
> **Ofc only after testing whether the new version works well on 2k3sp2 and in ros. Please post screenshots that prove both for the version that you suggest here.**
> 
> I do also contribute something: the proof by previous tests that both of our outdated existing versions do run in a comparable manner: search for "LibreOffice" in https://reactos.org/wiki/Tests_for_0.4.14
> I spotted 16bugs quickly in both. They do behave almost identically. A waste of time to always test 2 versions for each release.

Actually one of the two versions in ROS is already the latest that can run on XP/2003/Vista (Libreoffice 5.4.7.2).

I proposed to update the other one because 5.1.x is outdated if we consider that the last two branches of LibreOffice that supported XP/2003/Vista are 5.3.x and 5.4.x.

I think there are two version because of the Fresh/Still categorization of versions, but if only one should be kept, I would suggest to keep 5.4.7.2, as it is the last that can run on ROS and has already been tested (it's listed in the page you linked).

Just in case it might be needed, here is a screenshot of LibreOffice 5.3.7.2 on ReactOS 0.4.13 on Virtualbox 6.1.22 with Guest Additions and 3096 MB of RAM. 

ROS_LO

**Comment by JoachimHenze** [comment]
So we are at latest&greatest for our target already. Fine then.
Well then it's possible to delete the older one (5.1.1.3) just, if you want to update the PR for that. Otherwise we should just close this PR. I see no urgent need for any of the changes in libreoffice.txt.

**Comment by d1nuc0m** [comment]
@JoachimHenze updated the pull request.
Out of curiosity, why is there the CDPath parameter in LibreOffice file? (It is not documented in the wiki)

**Comment by d1nuc0m** [comment]
Probably not useful

**Pull request closed without merging**